content/es/multitenancy.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Multitenencia
```

Comment:
Hi @mdf-ido ! Can you send me some references about the term "Multitenencia", I couldn't find it. 
I found the term "Tenencia múltiple". We can mention the word "multitenancy" in English too in the text. 

- https://es.wikipedia.org/wiki/Tenencia_m%C3%BAltiple
- https://www.cloudflare.com/es-es/learning/cloud/what-is-multitenancy/
- https://platzi.com/blog/multi-tenant-que-es-y-por-que-es-importante/

Reply:
Hi @krol3 ! I was going off the wikipedia link you have listed here and then I found a link from IBM with the term: https://www.ibm.com/docs/es/xiv-storage-system?topic=STJTAG/com.ibm.help.xivgen3.doc/Gen3/PO/xiv_gen3_po_multi_tenancy.htm

I found the word to be strange as well since the term closest to it is tenant relating to housing, etc. 

Should I change it to Tenencia múltiple and use the english term in the text?

Reply:
Encontré una [opción de Google](https://cloud.google.com/identity-platform/docs/multi-tenancy-quickstart) que sería "multiusuario", aunque siento que se aleja bastante del concepto y mezcla con otros conceptos no relacionados y otra de [Microsoft](https://learn.microsoft.com/es-es/microsoft-365/admin/multi-tenant/manage?view=o365-worldwide) como "multiinquilino" que se acerca más al concepto de seguridad y aislamiento. Por último, la traducción de [RedHat](https://www.redhat.com/es/topics/cloud-computing/what-is-multitenancy) como "multiempresa", la cual me parece no cumple el 100% de los casos (por ejemplo, una misma empresa con varios proyectos), en fin creo que podemos discutirlo en la junta semanal.

Reply:
@mdf-ido @electrocucaracha  , Yo creo que podemos colocar la traducción más usada en la comunidad, y después usar la palabra en inglés, es bastante auto descriptiva y seria como otros casos de palabras técnicas como: pod, autoscaling, etc.

Reply:
Estaba buscando en Google Trends para ver si hay búsquedas a estos términos y la unica q dio resultados fue Tenencia múltiple. 

Recomiendan entonces usar: Tenencia Múltiple como título y multi-tenancy en las descripciones? 

[Trends](https://trends.google.com/trends/explore?q=Tenencia%20multiple)

Reply:
Hola @mdf-ido, podrías usar el término de "Tenencia Múltiple (multitenancy)" al inicio y ya las demás referencias usar multitenancy.